### DISCUSSION TITLE
Is there an easy way to add custom code to Head

### DISCUSSION THREAD

**Discussion by kb-1** [GitbookIO/community#500] [0. General bugs and questions]
Is there any easy way to add custom code to the Head of each page? I want to use a 3rd party tool that requires this, isnt already an integrated partner, and I am a non-technical user.

With most WYSIWYG website platforms (e.g Wix), this functionality is built in and its a few button clicks to add any custom code to the head that you'd like. But the only way I can see to do with with Gitbook is to create my own integration (which is beyond my non-technical user skills).  TIA!

**Comment by addisonschultz** [comment]
Hi @kb-1, thanks for the feedback - what integration/tool are you interested in using? It might be a good one to add as a feature request here in case there are others who are looking for your tool too! https://github.com/GitbookIO/integrations

Right now, an integration does need to be built to leverage the functionality you're asking
